bin/openupgrade/openupgrade.py

```diff
@@ -32,4 +32,4 @@
 
 _modules = __import__(_new_name, globals(), locals(), ['*'])
 for _i in dir(_modules):
-    locals()[_i] = getattr(_modules, _i)
+    locals()[_i] = getattr(_modules, _i)
```

Comment:
Why do you remove the last line here? Anyway, I think this is not needed anymore as you have replaced all the imports.